packages/pds/src/sql-repo-storage.ts

```diff
@@ -158,6 +158,7 @@ export class SqlRepoStorage extends RepoStorage {
           .selectFrom('repo_commit_block')
           .selectAll()
           .where('repo_commit_block.creator', '=', this.did)
+          .where('repo_commit_block.commit', '=', rebase.commit.toString())
```

Comment:
I like that this also makes the query more specific and safer.  It should be the same resultset with or without this condition, but if for some reason it's not, this is more specific and would delete fewer blocks 👍

Reply:
no it would actually delete more blocks i believe - this is a clause in a WHERE NOT EXISTS

Reply:
Oh yup of course.  Still seems all good to me.